examples/3D-pol-to-birefringence.py

```diff
@@ -76,17 +75,21 @@
 print(f"Shape of 3D phase data: {np.shape(phase3D)}")
 
 ## Save to zarr
-# Save birefringence
-writer = WaveorderWriter("./output")
-writer.create_zarr_root("reconstructions_" + timestamp)
-writer.init_array(
-    position=0,
-    data_shape=(1, 5, Z, Y, X),
-    chunk_size=(1, 1, 1, Y, X),
-    chan_names=["Retardance", "Orientation", "BF - computed", "DoP", "Phase"],
-)
-writer.write(birefringence, p=0, t=0, c=slice(0, 4), z=slice(0, Z))
-writer.write(phase3D, p=0, t=0, c=4, z=slice(0, Z))
+with open_ome_zarr(
+    "./output/reconstructions_" + timestamp,
+    layout="fov",
+    mode="w",
+    channel_names=[
+        "Retardance",
+        "Orientation",
+        "BF - computed",
+        "DoP",
+    ],
+) as dataset:
+    # Write to position "0", with length-one time dimension
+    dataset["0"] = birefringence[np.newaxis, ...]
+    dataset.append_channel("Phase")
+    dataset["0"][0, -1] = phase3D
```

Comment:
Am I using this correctly?

Reply:
Would `concatenate/stack` create an unnecessary copy? Maybe you have a different solution in mind?

Reply:
Good point. While in certain high-latency setup resizing the array is not necessarily faster than in-RAM copying since it needs at least one extra file writing operation (`.zarray`), it will definitely scale better if the arrays are very large. So in this case the fastest way might be creating zero-filled array first and fill the values.

Reply:
Great! I changed this to use `create_zeros`. (accidentally committed with a different bug fix in [a0dedde](https://github.com/mehta-lab/recOrder/pull/331/commits/a0dedde2f7641ebc2cd168c662a606a0a064726e))